content/getting-started/sections/installing-atom.md

```diff
@@ -159,6 +159,41 @@ $ sudo zypper in -y atom.x86_64.rpm
 
 {{/linux}}
 
+#### Updating Atom
+
+You should consider updating Atom periodically for the latest improvements to the software. Additionally, when Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.
```

Comment:
Sorry, I gave it one last read through in the context of the whole page and I think we don't need this sentence:

```suggestion
You should consider updating Atom periodically for the latest improvements to the software. Additionally, when Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible.
```

I think it's leftover from when this was initially proposed as a separate page.

Once we remove that I'll go ahead and merge.

Reply:
@rsese I agree, that just sounded awkward from a pig picture standpoint. I removed that sentence.